dcs/urls.py

```diff
@@ -9,7 +9,7 @@
 
     url(r'^$', 'dcs.views.home', name='home'), # mock, CHANGE later!
     url(r'^admin/', include(admin.site.urls)),
-    url(r'^multimedia/', include('multimedia.urls')),
+    url(r'^multimedia/', include('multimedia.urls', namespace='multimedia')),
```

Comment:
Not sure if I am being anal, but is it possible to omit the `multimedia/` in our url? user may play around with the URL, and find that he lands on a 404 page when he enters the URL 'ourdomain.com/multimedia` (which was what happened to me lol)
If you think what I asked is rather unreasonable just say uh haha